osmosis-1/osmosis.zone_assets.json

```diff
@@ -5274,6 +5274,97 @@
       "osmosis_unlisted": true,
       "listing_date_time_utc": "2024-10-20T12:00:00Z",
       "_comment": "Stratos native token (STOS)"
+    },
+    {
+      "chain_name": "int3face",
+      "base_denom": "uint3",
+      "path": "transfer/channel-82819/uint3",
+      "categories": [
+        "bridges"
+      ],
+      "osmosis_verified": false,
+      "listing_date_time_utc": "2024-10-11T03:00:00Z",
+      "_comment": "Int3face token"
+    },
+    {
+      "chain_name": "int3face",
+      "base_denom": "factory/int31zlefkpe3g0vvm9a4h0jf9000lmqutlh99h7fsd/dogecoin-doge",
+      "path": "transfer/channel-82819/factory/int31zlefkpe3g0vvm9a4h0jf9000lmqutlh99h7fsd/dogecoin-doge",
+      "osmosis_verified": false,
+      "transfer_methods": [
+        {
+          "name": "Int3face Bridge",
+          "type": "external_interface",
+          "deposit_url": "https://app.int3face.zone",
+          "withdraw_url": "https://app.int3face.zone"
+        }
+      ],
+      "canonical": {
```

Comment:
Please remove the canonical property (from all assets) since this hasn't (yet) been canonicalized.

Reply:
Removed